quadkey/library/js/tilecover.js

```diff
@@ -0,0 +1,585 @@
+(function() {
```

Comment:
Where is the copyright, license and notes about where this comes from? We don't have to add other stuff to files that come from other repositories if we copy them as-is, but we should be extremely clear about it in the license notices and document where they come from, what they do and how to update them.